lua/lspconfig/server_configurations/tailwindcss.lua

```diff
@@ -25,6 +25,7 @@ return {
       'handlebars',
       'hbs',
       'html',
+      'htmlangular',
       -- 'HTML (Eex)',
```

Comment:
can you also remove these comments lines.

Reply:
sure